src/App.css

```diff
@@ -71,13 +104,15 @@ ul {
 
 .display {
   width: 100%;
-  height: 70px;
+  height: 15%;
   background: var(--text-area);
 }
 
```

Comment:
- [OPTIONAL] The display of your calculator is not proportionate (it does not match) with the width of your calculator, it goes out of proportion as you resize the page, please kindly fix the issue, as it is a needed skill for a front end developer to stick to the design as much as possible.